testsuite/python-deep/src/test_deep.py

```diff
@@ -68,7 +60,7 @@ def print_deep_image (dd, prefix="After init,") :
                 print ()
 
 
-def print_deep_imagebuf (buf, prefix) :
+def print_deep_imagebuf (buf: oiio.ImageBuf, prefix: str) :
```

Comment:
Note to mostly-self: we should remember to take a pass over the docs and any python examples in the docs and change all the function declarations to use modern python3 syntax with type annotations.

Reply:
Just added this: https://github.com/AcademySoftwareFoundation/OpenImageIO/issues/4752